### PR TITLE
Harmonize left menu headings with generic names (#938)

### LINKED ISSUE #930: [DISCUSSION]: Standardization of Bitcoin and Lightning sub-headings: generic names vs program names
## Introduction

As stated by @Stadicus in this [comment](https://github.com/raspibolt/raspibolt/pull/921#issuecomment-1044668250), it is preferrable to have generic names for the left menu headings:

> Regarding "Generic" vs. "Application-specific" titles: I strongly prefer the generic titles, at least when there's a choice. This also affects the filenames, and we can replace individual applications in the core guide easier if necessary.

It does indeed make the guide more future-proof. However, presently, there are still a couple of headings that derive their names from the chosen program to install on the node, namely:
* Bitcoin Core
* LND
* Ride The Lightning

NB: Interestingly I assume "Electrum server" has now become a generic name! E.g. Fulcrum could be categorized as an Electrum server, correct? I looked around to see if there was a more generic name without the term 'Electrum' in it but coulnd't find anything at all. Any thoughts on this? Feel freet to bring this in the discussion below and propose alternative names.

NB 2: In this post, I'll focus on the "Bitcoin" and "Lightning" chapters and will ignore the "Raspberry Pi" chapter of the guide, although the heading "Raspberry Pi" is more specific than generic... Feel freet to bring this in the discussion if you wish so.

---

## New names proposals

I propose below a few ideas to homogenize the left menu headings with the following goals:
* *Generic rather than specific*: All headings are have generic names and not program names
* *Industry standard*: The generic name are well-known and accepted categories in the bitcoin community
* *Short*: The names are kept short for ease of display and navigation

### Program: Bitcoin Core

* _Existing_: **Bitcoin Core**
* _Proposed_:  **Bitcoin Node**
* _Alternative_: **Bitcoin Node Implementation** -> Quite long (also not an implementation per se as there is no specifications, correct?.. but that's pedantry now :)
* Alternative: **Bitcoin Client** -> but it's more than just a client...
* Alternative: **Bitcoin Core** -> we keep it as is, see [comment](https://github.com/raspibolt/raspibolt/pull/921#issuecomment-1044668250). IMHO, I don't think it's needed to keep the program name as the generic names above looks suitable.

### Program: Electrs

* _Existing_: **Electrum server** -> 
* _Proposed_: **Electrum Server** -> Keep existing name. Unless someone can come up with a more generic name that does not use the term 'Electrum', if that even exists.

### Program: BTC RPC Explorer

* _Existing_: **Blockchain Explorer** 
* _Proposed_: **Blockchain Explorer** -> Keep existing name.

### Program: LND

* _Existing_:  **LND**
* _Proposed_: **LN Node** -> Reuse the same terminology as 'Bitcoin Node' = visually consistent
* _Alternative_: **LN Node Implementation** -> Technically better, but quite long

### Program: Ride The Lightning

* _Existing_: **Ride The Lightning**
* _Proposed_: **Web Node Manager** -> The RTL repo describes itself as "a full function, device agnostic, web user interface to help manage lightning node operations", while ThuderHub has "an open-source LND node manager where you can manage and monitor your node on any device or browser"
* _Alternative_: **Web Dashboard** -> A bit simpler and shorter than the above... but nor RTL or TH use the term dashboard

### Program: Zeus

* _Existing (soon! with Zeus)_: **Mobile Wallet** -> not ideal as Zeus is much more than just a LN wallet
* _Proposed_: **Mobile Node Manager** -> to reuse the terminology used for RTL
* _Alternative_: **Mobile Dashboard** -> idem but with dashboard which is simpler/shorter

I'll update the OP with any comments, pros/cons, and new name proposals made in the comments below!
I'll add new sections if you want to discuss other names not listed just above (Electrum server, Raspberry Pi or something else).

Wdyat?

### PR DESCRIPTION
**Pull request by Stadicus**
Follow-up change to https://github.com/raspibolt/raspibolt/pull/938#event-6131425079

Original change description by @VajraOfIndra 

---

#### What

See discussion in #930 

_tl;dr_: The PR updates the left menu headings to make them generic rather than program-specific.

### Why

* Consistency/harmonization
* Makes it easier to switch a program with another one in the future if desired, without changing the guide's main structure

### How

* Changed file names: `*.md`
* Changed yml titles
* Changed h1 titles
* Changed the title/path of the "Next" links at the bottom of the page

### Scope

- [x] significant change to core configuration
- [ ] independent bonus guide
- [ ] simple bug fix

Fixes #930 

#### Test & maintenance

Make sure there is no broken links and that all relevant files have been updated with the proper name.

names

### CODE REVIEW

**Comment by Stadicus** [comment]
I cherry-picked my commit and added it on top of your original change. :)
https://www.atlassian.com/git/tutorials/cherry-pick

File: web-app.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: default
-title: Ride The Lightning
+title: Web app
 nav_order: 50
```
Comment (VajraOfIndra) [inline web-app.md:4]:
```suggestion
nav_order: 30
```

**Review by VajraOfIndra — APPROVED** [review]
That's a `lot` of bug fixes at once, that's excellent!

utACK!

I approved the PR, however if you have time I made one small comment (changing the nav_order ot web-app.md from 50 to 30).

**Comment by Stadicus** [comment]
I think once we merge this PR we should immediately start working on putting the individual files into the proposed folder structure. That includes a whole new round of link checking. 

@VajraOfIndra, do you want to give that a go as well? I don't have the time right now, so I'm a bit hesitant to just merge this now.

**Comment by VajraOfIndra** [comment]
> I think once we merge this PR we should immediately start working on putting the individual files into the proposed folder structure. That includes a whole new round of link checking.
> 
> @VajraOfIndra, do you want to give that a go as well? I don't have the time right now, so I'm a bit hesitant to just merge this now.

I'll give it a go. What about I do it as a PR to this raspibolt/raspibolt/generic-names branch?

**Comment by Stadicus** [comment]
> What about I do it as a PR to this raspibolt/raspibolt/generic-names branch?

Ah, that's a good option as well. No constant changes, one bit switch on merge to master and we're done. Sounds good, thanks!

**Comment by VajraOfIndra** [comment]
> > What about I do it as a PR to this raspibolt/raspibolt/generic-names branch?
> 
> Ah, that's a good option as well. No constant changes, one bit switch on merge to master and we're done. Sounds good, thanks!

I didn't manage to make a branch in my repo so I created a new one in the RaspiBolt repo. It's based on this branch (so with the file names changed): https://github.com/raspibolt/raspibolt/tree/generic-names+new-file-structure
* Edited the paths of files to reflect the structures discussed in #938 
* Added a number of missing {:target="_blank"} after external links
* Edited all internal links to work with the new structure

Pretty sure I've missed many links that will be broken though, there are a lots of them! And I couldn't use Dr Link Check
Also, I couldn't check the rendering of the left bar menu on GitHub Pages so I will let one of you do this as part of the review. It's possible that we'll have to fix some `nav_order` numbers.

I created PR to merge the new structure branch into this branch here with the new names.

**Comment by VajraOfIndra** [comment]
> I rebased the current branch (after merging in #938) onto the master branch to resolve any conflicts. A few additional changes were necessary (see commit [2fd3a6c](https://github.com/raspibolt/raspibolt/commit/2fd3a6c8a866f6ef42accce3b883ac69b357d446)), due to many moving parts and other changes that have been merged in the meantime.

:clap:  

> Now, all should be fine.
> 
> I would like to merge this change myself so that I can squash all commits manually and write a proper commit message. So many structural changes deserved a proper explanation for historians to dig through... 8-)

> Anyone wants to do a final check?

Is the branch visible on your own repo GitHub Page to do a final visual check..?

**Comment by VajraOfIndra** [comment]
> Yes, it's currently published to https://stadicus.github.io/raspibolt-dev/

Thanks. I had a quick look, checked the left menu and the main links that allow to move from one section to the next, also checked some of the pics.. it all looks good. I tried to plug the website in Dr Link Check but looks like I've reached my quota of allowed requests! 

Anyways, LGTM and ready to be merged into master.

**Review by VajraOfIndra — APPROVED** [review]

**Pull request merged**